Framework/Algorithms/src/SphericalAbsorption.cpp

```diff
@@ -64,7 +64,8 @@ void SphericalAbsorption::init() {
                   "The number density of the sample in number of atoms per "
                   "cubic angstrom, if not set with SetSampleMaterial");
   declareProperty("SphericalSampleRadius", EMPTY_DBL(), mustBePositive,
-                  "The radius of the spherical sample in centimetres");
+                  "The radius of the spherical sample in centimetresif "
```

Comment:
Small nit: missing space between `centimetres if`

Also I think this needs rewording slightly, to something like:
"in centimetres. This can be omitted if a spherical sample with known radius is set using SetSample`

Reply:
Thanks for spotting the typo, I think actually the doc string should reflect the fact that any value added there will be used instead of the sample radius on the shape, even if a valid one exists - I will do this.